news/2022-03-22-community-contributors-elite-nominators.md

```diff
@@ -0,0 +1,127 @@
+
+During this time, they almost single-handedly revived the venerable drum and bass genre in Ranked, catapulting many now astronomically popular maps such as [The Words I Never Said](https://osu.ppy.sh/beatmapsets/1470072) and [USSEEWA](https://osu.ppy.sh/beatmapsets/1322727) into the limelight for many to enjoy.
+
+In the words of the NAT, one could say that Zer0- is both drummed and based.
+
+## osu!taiko
+
+[![](/wiki/shared/news/2022-03-22-community-contributors-elite-nominators/Hivie.jpg)](https://osu.ppy.sh/users/14102976)
+
+Making his debut in the Beatmap Nominators during January 2021, [**Hivie**](https://osu.ppy.sh/users/14102976) quickly proved himself to be one of the most dedicated nominators the mode has possibly ever seen, reaching over 160 nominations with an excellent incident ratio in a very short time.
+
+Not merely content with his top-tier activity levels, Hivie took the initiative to get involved in many projects within the modding and mapping community, such as assisting with the Community Mentorship program, moderating the osu!taiko Modding & Mapping Hub Discord server, assisting Project Loved in multiple domains and now even managing the returning Beatmap Spotlights. Hivie has been simply *everywhere* at this point, and has continued to excel in all of it.
+
+[![](/wiki/shared/news/2022-03-22-community-contributors-elite-nominators/ikin5050.jpg)](https://osu.ppy.sh/users/4007649)
+
+Exploding into the Beatmap Nominators in late 2020, [**ikin5050**](https://osu.ppy.sh/users/4007649) is another example of stunning dedication to the scene with over 140 nominations made throughout his tenure, all showcasing a deep commitment to top-level modding quality and a great desire to push forward the future of mapping in osu!taiko.
+
+From his time as a mentee in the Community Mentorship program, ikin5050 has come a long way, having since taken mentees of his own and now reached the pinnacle of achievement in the Beatmap Nominators, something he more than deserves given his integral place in the osu!taiko community overall.
+
+## osu!catch
+
+[![](/wiki/shared/news/2022-03-22-community-contributors-elite-nominators/Secre.jpg)](https://osu.ppy.sh/users/2306637) 
+
+The most active osu!catch Beatmap Nominator of 2021 without contest, [**Secre**](https://osu.ppy.sh/users/2306637) has left a significant mark on the scene with his dedicated drive towards helping many mappers get their works situated in Ranked. 
+
+His outstanding drive for checking qualified mapsets has been instrumental in securing the ongoing quality of many osu!catch beatmaps, and his valuable insight and contribution towards osu!catch-specific Ranking Criteria discussions has been a significant boon for all. He has also since joined the Nomination Assessment Team to continue his efforts in their domain.
```

Comment:
very nitpicky but could use some variety
```suggestion
His outstanding ambition for checking qualified mapsets has been instrumental in securing the ongoing quality of many osu!catch beatmaps, and his valuable insight and contribution towards osu!catch-specific Ranking Criteria discussions has been a significant boon for all. He has also since joined the Nomination Assessment Team to continue his efforts in their domain.
```